public/components/Campaign/Campaign.js

```diff
@@ -43,6 +45,18 @@ class Campaign extends React.Component {
     } else { return (null); }
   }
 
+  renderReferralsIfGlobal(campaign) {
+    if ("global" === this.props.territory) {
+      return(<CampaignReferrals campaign={campaign} />);
+    } else { return (null); }
+  }
+
+  renderMediaEventsIfGlobal(campaign) {
```

Comment:
We will require territory specific data for media events. At the moment I only show the data we have for the global view.